src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -288,10 +301,16 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
       }
       case 'EvmCustomTx': {
         if (!inboundAddressData?.router) return undefined
+        if (!accountAssetAddress) return undefined
         const assetAddress = isToken(fromAssetId(assetId).assetReference)
           ? getAddress(fromAssetId(assetId).assetReference)
           : zeroAddress
-        const amount = BigInt(toBaseUnit(amountCryptoPrecision, asset.precision).toString())
+        const amount = // Reuse the savers util as a sane amount for the dust threshold
```

Comment:
Discussed this one out, slightly higher than LP dust thresholds but it's realistically $0.0003 vs. $0.0001 dust amount worst case scenario for BTC at current prices